src/main/kotlin/com/devfloor/untitled/articlehashtag/application/ArticleHashtagService.kt

```diff
@@ -11,6 +12,18 @@ class ArticleHashtagService(
     private val repository: ArticleHashtagRepository
 ) {
     @Transactional(readOnly = true)
-    fun showAllByArticle(article: Article): List<ArticleHashtag> =
-        repository.findAllByArticle(article)
+    fun showAllByArticle(article: Article): List<ArticleHashtag> {
+        return repository.findAllByArticle(article)
+    }
+
+    @Transactional
+    fun create(article: Article, hashtagList: List<Hashtag>): List<ArticleHashtag> {
+        val articleHashtagList = mutableListOf<ArticleHashtag>()
+        for(index in hashtagList.indices){
+            articleHashtagList.add(repository.save(ArticleHashtag(article, hashtagList[index])))
+        }
```

Comment:
이와 같이 변경하면 코드가 더 간단해 질 수도 있을거같아요 !!
```suggestion
        hashtagList.forEach {
            articleHashtagList.add(repository.save(ArticleHashtag(article, it)))
        }
```